ui_test/POM_hiway_asg/test_scripts/test_hiway_selenium.py

```diff
@@ -0,0 +1,203 @@
+import logging
+import pytest
+import csv
+import time
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') standard import "import time" should be placed before "import pytest" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=Pylint%3AC0411)